modules/yarn.lock

```diff
@@ -1081,6 +1081,15 @@
     "@clayui/layout" "^3.0.1"
     classnames "^2.2.6"
 
+"@clayui/alert@3.3.4":
```

Comment:
Not sure this is deduped correctly... 

> 369 changes: 129 additions & 240 deletions

This is usually a warning sign for me, uneven additions and deletions... in this case, surprisingly we deduped more than added, but there's a mix of new `@clayui` packages not deduped and other dependencies down the line `camelcase` that are in fact now better than they were... 🤔